libs/dh/admin/feature-user-management/src/lib/overview/dh-users-overview.component.ts

```diff
@@ -167,12 +170,8 @@ export class DhUsersOverviewComponent {
     this.store.reloadUsers();
   }
 
-  modalOnClose(): void {
-    this.isInviteUserModalVisible = false;
-  }
-
   showInviteUserModal(): void {
-    this.isInviteUserModalVisible = true;
+    this.inviteUserModal().openModal();
```

Comment:
suggestion: ideally we can use the `WattModalService` to open the component dynamically. Maybe something for a future PR.